Tests/Security/Authorization/Interception/MethodSecurityInterceptorTest.php

```diff
@@ -270,7 +270,10 @@ protected function getInterceptor(MetadataFactoryInterface $metadataFactory = nu
             ;
         }
 
-        $securityContext = $this->getMockBuilder('Symfony\Component\Security\Core\SecurityContext')
+        $securityContextClass = class_exists('Symfony\Component\Security\Core\SecurityContext')
+            ? 'Symfony\Component\Security\Core\SecurityContext'
+            : 'Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface';
```

Comment:
I agree but that would be complicated here as this function doesn't test anything and is just here to simplify the rest of the class.